test/tests/functional/pbs_cray_check_node_exclusivity.py

```diff
@@ -563,28 +563,42 @@ def test_job_running_on_multinode_reservation(self):
         """
         Test to submit job on multinode reservation with different placement
         """
+        ncpus = []
+        vnodes = self.server.status(NODE)
+        num_vnodes = 2
+        i = 0
+        for vnode in vnodes:
+            if i < 2:
+                if vnode['resources_available.vntype'] == 'cray_compute':
+                    ncpus.append(int(vnode['resources_available.ncpus']))
+                    i += 1
```

Comment:
Why don't you break when i == 2?  That way you don't complete the loop doing nothing.